.github/CODEOWNERS

```diff
@@ -0,0 +1,23 @@
+# This list auto requests reviews from the specified org members
+# when a PR that modifies the file in question is opened
+# This list is alphabetized by User -> Filename KEEP IT THAT WAY
+# In the event that multiple org members are to be informed of changes
+# to the same file or dir, add them to the end under Multiple Owners
+
+# Alterist
+
+/_maps/ @Alterist
+/icons/ @Alterist
+
+# Hughgent

```

Comment:
As I said before you can't have non members in the code owners file, it just doesnt work

Reply:
Oh I thought hugh was a maintainer for some reason